.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,24 @@
+name: Feature Request
+description: Suggest an idea for improving JPlag
+title: ""
+labels: ["enhancement"]
+assignees: []
+
+body:
+  - type: textarea
+    id: feature_description
+    attributes:
+      label: Feature Description
+      description: A clear and concise description of the feature you would like to see.
+      placeholder: E.g., add support for CSV export of similarity reports
```

Comment:
I would leave out the placeholders here. Or just have them state: Insert description/use case here...